documentation/modules/configuring/con-config-security-providers.adoc

```diff
@@ -0,0 +1,149 @@
+// Module included in the following assemblies:
+//
+// assembly-security-providers.adoc
+
+[id='con-config-security-providers-{context}']
+= How to configure security context
+
+[role="_abstract"]
+Use security provider plugins or template configuration to apply security context to Strimzi pods and containers.
+
+Apply security context at the pod or container level:
+
+Pod-level security context:: Pod-level security context is applied to all containers in a specific pod.
+Container-level security context:: Container-level security context is applied to a specific container.
+
+With Strimzi, security context is applied through one or both of the following methods:
+
+Template configuration:: Use `template` configuration of Strimzi custom resources to specify security context at the pod or container level.
+Pod security provider plugins:: Use pod security provider plugins to automatically set security context across all pods and containers using preconfigured settings.  
+  
+Pod security providers offer a simpler alternative to specifying security context through `template` configuration.
+You can use both approaches.
+The `template` approach has a higher priority.
```

Comment:
I was wondering if it makes sense to use the "override" word to explain that if you have a pod security provider set, its configuration could be "overridden" by what you put in the "template" (if you use it).

Reply:
It might make it clearer. I've added _Security context configured through `template` properties overrides the configuration set by pod security providers. _